common/servlet/src/main/java/io/apiman/common/servlet/ApimanCorsFilter.java

```diff
@@ -29,16 +31,23 @@
  */
 public class ApimanCorsFilter implements Filter {
 
+    public static final Configuration config;
+    static {
+        config = ConfigFactory.createConfig();
+    }
+
+    public static final String MANAGER_UI_ALLOWED_CORS_ORIGINS = "apiman-manager-ui.allowed-cors-origins";
+
     private HashSet<String> allowedCorsOrigins = new HashSet<>();
 
     /**
      * Constructor.
      */
     public ApimanCorsFilter() {
-        String corsOrigins = System.getProperty("allowed_cors_origins");
-        if (corsOrigins != null) {
-            allowedCorsOrigins = new HashSet<>(Arrays.asList(corsOrigins.trim().split(",")));
-        }
+        String corsOrigins = System.getProperty(MANAGER_UI_ALLOWED_CORS_ORIGINS,
+                                                config.getString(MANAGER_UI_ALLOWED_CORS_ORIGINS,
+                                                                "*"));
```

Comment:
So we allow all origins by default, if config is not set?

Reply:
That was probably the original behavior... :)

Reply:
So we keep it :)

Reply:
> So we allow all origins by default, if config is not set?

Yes